lib/cmd/fh3/appdata/import/status.js

```diff
@@ -0,0 +1,128 @@
+"use strict";
+/* globals i18n */
+
+var fs = require('fs')
+  , _ = require('underscore')
+  , fhreq = require('../../../../utils/request.js')
+  , async = require('async')
+  , ProgressBar = require('progress')
+  , common = require('../../../../common');
+
+var POLL_INTERVAL = 2000;
+var MIN_INTERVAL = 100;
+var EXIT_KEYS = ['q', 'Q'];
+
+/**
+ * Read progress information by calling the `read` endpoint to get the
+ * job definition. Progress information is included there:
+ * - step: current import job step
+ * - totalSteps: total import steps
+ */
+function readProgress(params, cb) {
+  var url = 'api/v2/appdata/' + params.envId + '/' + params.appId + '/import/data/' + params.jobId;
+  common.doGetApiCall(fhreq.getFeedHenryUrl(), url, 'Error reading progress', cb);
+}
+
+/**
+ * Update the progressbar if possible. Don't use `bar#tick` as that will not
+ * set the current progress but add the new valud to the current progress.
+ */
+function printout(data, bar) {
+  if (bar && data.totalSteps && data.step >= 0) {
+    bar.update(data.step / data.totalSteps);
+  } else {
+    /* eslint disable-next-line no-console */
+    console.log(i18n._('No progress information in response'));
+  }
+}
+
+/**
+ * Get a sane poll interval (use the default one if the user did not provide
+ * one)
+ */
+function getPollInterval(params) {
+  if (params.interval && _.isNumber(params.interval)) {
+    var interval = params.interval;
+    return interval >= MIN_INTERVAL ? interval : POLL_INTERVAL;
+  } else {
+    return POLL_INTERVAL;
+  }
+}
+
+module.exports = {
+  'desc': i18n._('Poll for job status'),
+  'examples': [{
+    cmd: [
+      'fhc addpata import status --appId=5ujx2eifvzaudq43nw4nmvcu --envId=live --jobId=5731da1dd3e2b283203c4054'
+    ].join(''),
+    desc: i18n._('Poll for job status')
+  }],
+  'demand': ['envId', 'appId', 'jobId'],
+  'alias': {},
+  'describe': {
+    'envId': i18n._('Environment id'),
+    'appId': i18n._('Application id'),
+    'jobId': i18n._('Import job id'),
+    'interval': i18n._('Polling interval in milliseconds (default is 2000)')
+  },
+  'method': 'get',
+  'customCmd': function (params, cb) {
+    var bar = null, poller = null;
+
+    function cleanupAndExit(err) {
+      if (poller) {
+        clearInterval(poller);
+      }
+      return cb(err);
+    }
+
+    // Need to use `console#log` here until we introduce some other logging or output
+    // framework to fhc
+    /* eslint disable-next-line no-console */
+    console.log(i18n._('Polling job progress. Press `q` to quit'));
+
+    function getProgressBar(job) {
+      if (bar === null && job.totalSteps) {
+        bar = new ProgressBar(i18n._('Job progress: [:bar] :current/:total :percent :elapseds :etas'), {
```

Comment:
Including the template strings in the translatable thing without the proper comments may confuses translators and might breaks accidentally. maybe good to separate them like i18n._('Job progress: ') + ...